code/modules/antagonists/revolution/revolution.dm

```diff
@@ -234,6 +236,8 @@
 
 	if((ishuman(owner.current)))
 		if(owner.current.stat != DEAD)
+			if(deconversion_reason == DECONVERTER_STATION_WIN)
```

Comment:
Move this up to the beginning of `farewell`, it has no reason to be this deep in the chain.